cylc/flow/scheduler.py

```diff
@@ -780,47 +780,42 @@ def process_queued_task_messages(self):
     def process_command_queue(self):
         """Process queued commands."""
         qsize = self.command_queue.qsize()
-        if qsize > 0:
-            log_msg = 'Processing ' + str(qsize) + ' queued command(s)'
-        else:
+        if qsize <= 0:
             return
-
+        LOG.info(f"Processing {qsize} queued command(s)")
```

Comment:
Unfortunately, getting the commands before actually running them would be a more complex change, so hopefully just logging the number of commands is ok